packages/volto/news/4171.bugfix

```diff
@@ -0,0 +1 @@
+Use a text widget for editing the frontend domain setting, instead of the URL widget which causes problems by erasing the current domain. @davisagli
```

Comment:
```suggestion
Use a text widget for editing the frontend domain setting in the Volto Settings control panel, instead of the URL widget which causes problems by erasing the current domain. @davisagli
```